### DISCUSSION TITLE
How to Install for Windows user and Docker Newbe

### DISCUSSION THREAD

**Discussion by mylogon1** [arabcoders/watchstate#405] [General] [answered]
I get lost on this and cannot find any tutorials on how to install when there is no container already made.

I apologize for such a basic request but Docker and I and strangers

**Comment by arabcoders** [comment]
Hello,

Sorry i am not sure i understand, the container is already pre-built, you copy over the docker-compose.yaml and run `docker compose up -d` 

If you are asking how to install docker on windows this video probably helps https://www.youtube.com/watch?v=cMyoSkQZ41E

**Reply by mylogon1** [reply]
I will try that. I saw the create the .yaml file and knew that was used in linux, so that's how I was confused. Thanks for your speedy response. I will try this later today.

**Reply by arabcoders** [reply, marked as answer]
No problem. Please update if you face any problem regarding the container.
